### ISSUE TITLE
Thumbnails mode with zoom on hover and thumbnail first if focused

### ISSUE THREAD

**Issue by hycday** [rcbyr/keen-slider#239]
hi all,
I am struggling to achieve the following :

If we take the "Thumbnails" example from https://keen-slider.io/examples in vanillaJS, how to achieve: 
- hovering the image currently dislayed in big, will zoom inside the image
- clicking on a thumbnails will move it to the far left (or far top if the view for thumbnails is vertical), taking into account that it's only possible if there are at least one more image after

in short : imagine this used for shop to diplay gallery of a product like https://fr.yeti.com/products/loadout-go-box-30?variant=38191954591923 for example

**Comment by rcbyr** [comment]
Hey @hycday 

it is a bit hard to answer this question. Have you tried to implement it? Do you have a working status that shows where the problem is?

**Comment by hycday** [comment]
@rcbyr sure :

here you can see my demo https://codesandbox.io/s/sparkling-sun-pv3fgf

Regarding my point "inline zoom on hovering": 
1- the inline zoom on hovering works fine (using another script), but is it acting weird : the image before hovering is actually only showing part of the image, once i hover, it does inline zoom, and once i hover out, then i see full image. 

2- I use the navigation and the keyboardControls plugins for keen, but a) the keyboardControls only works if i click on the thumbnails first, and b) the keyboardControls AND the navigation actually only move the thumbnails slider, but doesnt switch to next thumbnails/image

3- I would like to remove the dots from the navigation, but doing so will give me some errors

**Comment by rcbyr** [comment]
@hycday Something is wrong with your codesandbox. The files are empty. Can you fix this?

**Comment by hycday** [comment]
@rcbyr I was just about to do it, but then I read again my code and noticed my mistake...

1- was solved by adding following css
`    background-size: cover;
    background-position: center center;`

2- was solved by doing the trick of having the navigation plugin on the slider and not on the thumbnails

3- silly me didnt remove the 'dot' reference everywhere... now all good


so now i have "another" issue, to make it work vertically :

4- if I have "perView" = 3, then thumbnails after 3 aren't displayed
5- I am not sure how to change the navigation arrows to be on top and bottom rather than left/right (i know the svg are showing left/right icons, but just for the exercise i would like to place them up and down)

here is the codesandbox : https://codesandbox.io/s/sparkling-sun-pv3fgf

**Comment by pySilver** [comment]
@hycday do you have a working example from the link above by any chance? I'm looking for a very similar implementation. What zoom lib did you use?
